app/views/conditions/edit.erb

```diff
@@ -0,0 +1,78 @@
+<% @title = "Edit Weather Info" %>
```

Comment:
@tmikeschu you'll want to set any instance variables in the controller. in this case, i believe that's the only way to set the html head 👍